msteams-platform/samples/virtual-table-entity-references.md

```diff
@@ -0,0 +1,292 @@
+---
+title: Virtual table entity references
+author: surbhigupta
+description: In this module, learn about virtual tables entity reference and their attributes in Microsoft Teams.
+ms.localizationpriority: medium
+ms.author: v-npaladugu
+ms.topic: conceptual
+---
+
+# Virtual tables entity reference
+
+Learn about all the available Collaboration controls virtual entities and their attributes. These entities have a one-to-one mapping with a specific Microsoft Graph resource type, for example, the Graph Planner Task entities maps to the [Microsoft Graph Planner Task resource type](/graph/api/resources/plannertask). The virtual entity shares the same attributes as the resource type.
+
+> [!NOTE]
+> Currently Collaboration controls are available only in [public developer preview](~/resources/dev-preview/developer-preview-intro.md).
+
+## Collaboration controls virtual entities
+
+| Name | Description |
+|---|---|
+| Graph Planner Task | The Graph Planner Task table represents a Planner task in Microsoft 365. |
+| Graph Planner Plan | The Graph Planner Plan table represents a Planner plan in Microsoft 365. |
+| Graph Event | The Graph Event table represents an event in a user calendar, or the default calendar of a Microsoft 365 group. |
+| Graph Booking Appointment | The Graph Booking Appointment table represents a customer appointment for a Booking Service, performed by a set of staff members, provIDed by a Microsoft Bookings business. |
+| Graph Drive | The Graph Drive table represents the top-level object that represents a user's OneDrive or a document library in SharePoint. |
+| Graph Drive Item | The Graph Drive Item table represents a file, folder, or other item stored in a drive. |
+
+## Graph Planner Task
+
+* Entity name: m365_graphplannertask.
+* Graph resource: [plannerTask resource type](/graph/api/resources/plannertask)
+* Sorting isn't supported.
+* Filtering isn't supported.
+* Server driven pagination is supported, with maximum page size being 400.
+
+### Attributes for Graph Planner Task
+
+| Column  | Dataverse Type | Details |
+|---|---|---|
+| `m365_collaborationrootid` | StringType | Collaboration root ID of the collaboration session record is associated with multiple collaboration sessions. This will be returned as a comma delimited string. Note that this attribute won't be returned when retrieving multiple records. |
```

Comment:
Add this line

| `m365_graphplannertaskId` | guid | Unique identifier of the graph planner task. |